### PR TITLE
make downloading files back to server optional

### PR DESCRIPTION
**Pull request by brianlball**
Issue https://github.com/NREL/OpenStudio-server/issues/570

The defaults are to still download results.
The user can override.

### CODE REVIEW

File: server/app/jobs/dj_jobs/run_simulate_data_point.rb [hunk 1]
```diff
@@ -284,21 +284,35 @@ def perform
 
           # Post the reports back to the server
           uploads_successful = []
-
-          Dir["#{simulation_dir}/reports/*.{html,json,csv}"].each { |rep| uploads_successful << upload_file(rep, 'Report') }
-
+          if @data_point.analysis.download_reports
```
Comment (nllong) [inline server/app/jobs/dj_jobs/run_simulate_data_point.rb:287]: 👍

File: local_setup_scripts/rebuild_sr.sh [hunk 2]
```diff
@@ -6,23 +6,23 @@ docker volume rm osdata -f || true
 docker volume rm dbdata -f || true
 
 while [ $(docker ps -q | wc -l) != 1 ]; do sleep 5; done
-docker image rm 127.0.0.1:5000/openstudio-server -f
-docker build . -t="127.0.0.1:5000/openstudio-server" --build-arg OPENSTUDIO_VERSION=3.0.0-beta
+#docker image rm 127.0.0.1:5000/openstudio-server -f
```
Comment (nllong) [inline local_setup_scripts/rebuild_sr.sh:9]: was this change intentional?

File: server/app/models/analysis.rb [hunk 3]
```diff
@@ -54,7 +54,11 @@ class Analysis
   field :initialize_worker_timeout, type: Integer, default: 28800 # set default to 8 hrs
   field :upload_results_timeout, type: Integer, default: 28800 # set default to 8 hrs
   field :run_workflow_timeout, type: Integer, default: 28800  # set default to 8 hrs
-
+  field :download_zip, type: Boolean, default: true
```
Comment (nllong) [inline server/app/models/analysis.rb:57]: thanks for setting the defaults

File: local_setup_scripts/rebuild_sr.sh (hunk 2 above)
Comment (brianlball) [reply local_setup_scripts/rebuild_sr.sh:9]: yeah

**Comment by nllong** [comment]
changes look good! thanks for implementing this.
